client/src/ui/components/quest/steps/createDefenseArmySteps.tsx

```
@@ -125,80 +117,57 @@ export const createDefenseArmySteps: StepOptions[] = [
     },
-    classes: "-mt-5",
     advanceOn: {
       selector: ".defense-army-selector",
       event: "click",
     },
+    beforeShowPromise: function () {
+      return waitForElement(".defense-army-selector");
+    },
     showOn: () => {
       const showStep = document.querySelector(".defensive-army-selector");
       return !Boolean(showStep);
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)